.gitignore

```diff
@@ -58,7 +58,7 @@ coverage.xml
 
 .nyc_output/out.json
 coverage-cy
-PYTHON/blocks
+PYTHON/blocks*
```

Comment:
@itsjoeoui  `PYTHON/blocks*` is used to ignore any folder or file starts with block prefix as when updating existing blocks folder will renamed to `blocks_{timestamp}`.

If you are wondering why renaming old folder instead of wiping it out, I tried in Mac and Windows respectively and faced issue with permission when deleting complete folder. So to be safe and ignore errors I adopted this approach.